contents/blog/startup-ops-toolkit.md

```diff
@@ -0,0 +1,93 @@
+---
+date: 2021-03-23
+title: The Startup Ops Toolkit
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+---
+
+Ok, so you’ve read *Zero to One* and *The Hard Thing About Hard Things*, but what are all the boring admin things that are actually going to drain 40% of your time which no one has told you about?
+
+<iframe src="https://giphy.com/embed/pp1H7Z5TX3z44" width="480" height="245" frameBorder="0" class="giphy-embed" allowFullScreen></iframe><p><a href="https://giphy.com/gifs/reaction-cloudy-with-a-chance-of-meatballs-pp1H7Z5TX3z44">via GIPHY</a></p>
```

Comment:
I didn't get this gif! Rest were fine

Reply:
It was meant to be like a 'haha, oh wait' type gif but I can see it was much funnier in my head. 

How about this one: https://media.giphy.com/media/j6wvxJLueHV10lATCF/giphy.gif

Reply:
>  it was much funnier in my head

Agreed. New one is better!

Reply:
Done!